components/infobox/wikis/apexlegends/infobox_map_custom.lua

```diff
@@ -67,16 +72,17 @@ function CustomInjector:addCustomCells(widgets)
 				['padding-bottom'] = '0px',
 				['border-top-style'] = 'none'},
 			}
-		ringTable:addRow(CustomMap:_createRingTableHeader())
+		ringTable:addRow(CustomMap:_createRingTableHeader() --[[@as Html]])
```

Comment:
This the wrong, because the annotation for `addRow()` is wrong, the input should be `WidgetTableRow` not `Html`

Reply:
```suggestion
		ringTable:addRow(CustomMap:_createRingTableHeader())
```

Reply:
will add the anno fix in WidgetTableRow via different PR